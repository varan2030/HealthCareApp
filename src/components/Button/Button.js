import React, { Component } from 'react';

class Button extends Component {
  render() {
    return (
      <div
        className={ this.props.className }
        onClick={ this.props.toggleClassName }
      >
        { this.props.children }
      </div>
    )    
  }
}

export default Button