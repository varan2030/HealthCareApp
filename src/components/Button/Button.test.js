import React from 'react';
import Enzyme, {shallow, mount} from 'enzyme';
import Button from "./";
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({adapter: new Adapter()});

describe("Button component", () =>{
  
  test('renders', () => {
    const wrapper = shallow(<Button />);
    expect(wrapper.exists()).toBe(true);
  })

  test('when the button is clicked the menu is opened', () =>{
    const wrapper = (<Button />);
    expect(wrapper).toMatchSnapshot();
  })
});

