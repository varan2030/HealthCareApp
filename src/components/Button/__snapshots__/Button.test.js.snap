// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component when the button is clicked the menu is opened 1`] = `<Button />`;
